pkg/webhook/workspace/handler/immutable.go

```diff
@@ -81,8 +84,26 @@ func (h *WebhookHandler) handleImmutableRoute(oldObj, newObj runtime.Object, use
 	if username == h.ControllerSAName {
 		return true, ""
 	}
-	if reflect.DeepEqual(oldObj, newObj) {
+	if changePermitted(oldObj, newObj) {
```

Comment:
Does not go have the same func convention as Java like isChangePermitted?

Reply:
I don't think there's any such naming convention, but Go generally tries to avoid the verbosity of Java :)